app/helpers/navigation_items.rb

```diff
@@ -86,7 +86,7 @@ def for_provider_account_nav(current_provider_user, current_controller, performi
       end
 
       sign_out_navigation = if current_provider_user.impersonator
-                              NavigationItem.new('Return to support',
+                              NavigationItem.new('Support',
```

Comment:
Not directly related, but a defect from the Organisation Settings PR -- this does not affect real users, only support users impersonating a provider user. When the 'Organisation settings' link is shown, the navbar line is too long and it wraps. Shortening this text fixes it.